src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialService.scala

```diff
@@ -259,58 +259,78 @@ final class TrialService
     }
   }
 
-  private def enrollUserInternal(userInfo: UserInfo, status: UserTrialStatus): Future[PerRequestMessage] = {
+  private def enrollUserInternal(userInfo: UserInfo, enabledStatus: UserTrialStatus): Future[PerRequestMessage] = {
```

Comment:
This function was already a bit of a monster before this PR, might be nice to break out the block that creates `statusFuture`

Reply:
done!